modules/ROOT/pages/bat-command-reference.adoc

```diff
@@ -1,23 +1,52 @@
 = BAT CLI Reference
 
-The BAT command line interface lets you run tests locally and schedule monitors for continuous API testing. You can schedule tests in the cloud using the CLI-xAPI.  The BAT CLI is integrated with Anypoint Platform.
+BAT CLI provides an API Functional Monitoring command line interface through an integration with Anypoint Platform. The command line interface lets you run tests locally and schedule monitors for continuous API testing. You can schedule tests in the cloud using the CLI-xAPI. 
 
-This reference covers the syntax and key options you can use with the *bat* command. Using the *bat* command, you can debug and run tests, get credentials for scheduling tests, schedule tests, list test schedules, and perform additional operations.
+This reference covers the syntax and key options you can use with the `bat` commands. Using the `bat` commands, you can debug and run tests, get credentials for scheduling tests, schedule tests, list test schedules, and perform additional operations.
 
-== bat Command Syntax
+For commands to execute successfully, the account under which BAT CLI logs in must have the required permissions for the action the command is executing. 
 
-The `bat` command provides an API Functional Monitoring command line interface.
+== Permissions Required
 
-`bat [<file> ... | <option> ...]`
+To use all of the `bat` commands to execute the corresponding actions in Anypoint Platform, you must have the following permissions. 
+
+[options="header,footer"]
+|=======================
+|Permission |Actions
+|Create Applications (Runtime Manager) |Create private locations
+|Delete Applications (Runtime Manager) |Delete private locations
+|Manage Settings (Runtime Manager) |Update private locations
+|View Environment |Log in using BAT CLI
+|View Organization |Use Secrets Manager through BAT CLI
+|Exchange Contributor (Exchange)| Create or modify a monitor; add or modify a schedule
+|Exchange Administrator (Exchange)| Create, modify, or delete a monitor; add, modify, or delete a schedule 
+|Grant access to secrets (Secrets Manager) |Use Secrets Manager
+|=======================
+
+== Login Options
+
+To use the `bat` commands to execute API Functional Monitoring actions, you must first log in using one of the following login methods. 
+
+=== Login Using Connected Apps
+
+`bat [login --client-id --client-secret [--profile=<value>] --host=<value>]`
+
+This option logs into an Anypoint Platform account using the client ID and client secret from a connected app. This provides an alternative way to log in using MFA (multi-factor authentication), so that a second factor is not required. MFA is now required for non-SSO accounts unless they are on your organization's exemption list. For information on connected apps, which are configured by organization administrators, see xref:access-management::connected-apps-overview.adoc[Connected Apps].
```

Comment:
```suggestion
This option logs in to an Anypoint Platform account using the client ID and client secret from a connected app. This provides an alternative way to log in using MFA (multi-factor authentication), so that a second factor is not required. MFA is now required for non-SSO accounts unless they are on your organization's exemption list. For information on connected apps, which are configured by organization administrators, see xref:access-management::connected-apps-overview.adoc[Connected Apps].
```